cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1 on 2024-12-28 10:33
+
+import cinema.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="movie",
+            name="image",
+            field=models.ImageField(
+                null=True, upload_to=cinema.models.get_upload_image_path
```

Comment:
Ensure that the `get_upload_image_path` function in `cinema.models` is correctly defined to format the image path as required: `f"{slugify(movie.title)}-{uuid}{ext}"`. This is crucial for meeting the task's requirement for image saving format.